test/e2e/scale.go

```diff
@@ -168,6 +172,11 @@ func scaleSpec(ctx context.Context, inputGetter func() scaleSpecInput) {
 			LogFolder:           filepath.Join(input.ArtifactFolder, "clusters", input.BootstrapClusterProxy.GetName()),
 			IgnoreAlreadyExists: true,
 		})
+
+		if input.PostNamespaceCreated != nil {
```

Comment:
Note: this is the only test that is not using setupSpecNamespace (the reason is explained above; we are pinning namespace name vs creating a name with a random suffix).